src/uucore/src/lib/features/format/escape.rs

```diff
@@ -99,13 +113,16 @@ fn parse_unicode(input: &mut &[u8], digits: u8) -> Option<char> {
 pub struct EscapeError {}
 
 /// Parse an escape sequence, like `\n` or `\xff`, etc.
-pub fn parse_escape_code(rest: &mut &[u8]) -> Result<EscapedChar, EscapeError> {
+pub fn parse_escape_code(
+    rest: &mut &[u8],
+    zero_octal_parsing: Option<OctalParsing>,
```

Comment:
I'm not sure about the `Option` here. `None` indicates that the default variant of `OctalParsing` should be used (and not that there is no `OctalParsing`). Maybe it makes sense to explicitly mark one of the `OctalParsing` variants as default and use `OctalParsing::default()` where `None` is currently used?